.github/workflows/BuildPR.yml

```diff
@@ -0,0 +1,27 @@
+name: PullRequest
```

Comment:
```suggestion
name: build-gitx
```

Reply:
Then https://github.com/gitx/gitx/workflows/build-gitx/badge.svg should work I think